### PR TITLE
Resolves #1541

### LINKED ISSUE #1541: Set-WebApplicationProxyConfiguration is Missing Some Parameter Descriptions
In the `WebApplicationProxy` module, the following `Set-WebApplicationProxyConfiguration` parameters are missing descriptions:

- ADFSTokenAcceptanceDurationSec
- UserIdleTimeoutAction
- UserIdleTimeoutSec

### PR DESCRIPTION
**Pull request by velkovb**
Add missing parameter description
https://github.com/MicrosoftDocs/windows-powershell-docs/issues/1541

### CODE REVIEW

**Comment by e0i** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Comment by e0i** [comment]
@dariomws306 @tseward

Peer review needed for this PR. Thanks.

File: docset/windows/webapplicationproxy/Set-WebApplicationProxyConfiguration.md [hunk 1]
```diff
@@ -272,7 +272,7 @@ Accept wildcard characters: False
 ```
 
 ### -UserIdleTimeoutSec
-
+Specifies the inactivity time interval, in seconds, after which Web Application Proxy will redirect user to AD FS.
```
Comment (JohanFreelancer9) [inline docset/windows/webapplicationproxy/Set-WebApplicationProxyConfiguration.md]:
```suggestion
Specifies the inactivity time interval, in seconds, after which Web Application Proxy will redirect users to AD FS.
```

File: docset/windows/webapplicationproxy/Set-WebApplicationProxyConfiguration.md [hunk 2]
```diff
@@ -66,7 +66,7 @@ Accept wildcard characters: False
 ```
 
 ### -ADFSTokenAcceptanceDurationSec
-
+Specifies the maximum duration in seconds until which the Web Application Proxy server accepts the edge token issued by the AD FS server.
```
Comment (JohanFreelancer9) [inline docset/windows/webapplicationproxy/Set-WebApplicationProxyConfiguration.md]:
```suggestion
Specifies the maximum duration in seconds until when the Web Application Proxy server accepts the edge token issued by the AD FS server.
```

**Pull request merged**